Sources/AWSLambdaRuntimeCore/Documentation.docc/Resources/code/03-02-03-package.swift

```diff
@@ -0,0 +1,14 @@
+// swift-tools-version:5.8
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "SquareNumberLambda",
+    platforms: [
+        .macOS(.v12),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/swift-server/swift-aws-lambda-runtime.git", .upToNextMajor(from: "1.0.0-alpha")),
```

Comment:
can simplify this to `from: "1.0.0-alpha"`

Reply:
done https://github.com/swift-server/swift-aws-lambda-runtime/pull/300/commits/54ff67c394e8b37a70142bfcf21b80ececedcbd4